packages/shared/styles/components/atoms/SfQuantitySelector.scss

```diff
@@ -1,22 +1,76 @@
 @import "../../helpers";
 
 .sf-quantity-selector {
-  --input-width: 1.875rem;
-  --input-height: 1.6875rem;
-  input {
-    --input-padding: 0;
-    --input-margin: 0;
-    --input-border-width: 1px;
-    text-align: center;
-    transition: border-color 150ms linear;
-    &:focus {
-      --input-border: 1px solid var(--c-primary);
-      --input-border-width: 1px 1px 0 1px;
-      --input-padding: 0 0 1px 0;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  height: var(--quantity-selector-height, 3.5rem);
+  width: var(--quantity-selector-width, 23.4375rem);
+  padding: var(--quantity-selector-padding, 0);
+  background: var(--quantity-selector-background);
+  --button-color: var(--c-text);
+  --button-background: var(--c-light);
+  --button-padding: 0;
+  --button-size: 1.625rem;
+  --input-margin: 0;
+  --input-padding: 0;
+  --input-border: none;
+  --input-height: 3.5rem;
+  &__button {
+    border-radius: var(--quantity-selector-button-border-radius, 50%);
+    &:hover {
+      --button-background: var(--c-light);
+      box-shadow: none;
     }
   }
+  &__input {
+    .sf-input__wrapper input {
+      text-align: center;
+    }
+    .sf-input__bar::after,
+    .sf-input__bar::before {
+      display: none;
+    }
+  }
+  &::before {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 1px;
+    background: var(--c-light);
+    transition: none;
+    content: "";
+  }
   @include for-desktop {
-    --input-width: 3.125rem;
-    --input-background: var(--c-light);
+    --quantity-selector-background: var(--c-light);
+    --quantity-selector-width: 6.75rem;
+    --quantity-selector-height: 3.25rem;
+    --quantity-selector-padding: 0 0.625rem;
+    --quantity-selector-button-border-radius: 0;
+    --quantity-selector-font-size: 22px;
+    --button-height: var(--quantity-selector-height, auto);
+    &__button {
+      &:hover {
+        text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
here, you should use also variable for color from base colors: `var(--dark-base)`